### PR TITLE
Expose additional variables

### PR DESCRIPTION
**Pull request by molel-gt**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ flake8`
- [ ] All tests pass: `$ python run-tests.py --unit`
- [ ] The documentation builds: `$ cd docs` and then `$ make clean; make html`

You can run all three at once, using `$ python run-tests.py --quick`.

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py [hunk 1]
```diff
@@ -367,6 +368,10 @@ def __init__(self, name="Doyle-Fuller-Newman half cell model", options=None):
             + pot * vdrop_cell,
             "Terminal voltage": voltage,
             "Terminal voltage [V]": phi_s_w_ref + param.U_n_ref + pot * voltage,
+            "Instantaneous power [W.m-2]": i_cell * i_typ * (phi_s_w_ref + param.U_n_ref + pot * voltage),
+            "Pore-wall flux [mol.m-2.s-1]": j_w,
+            "Ratio of electrolyte diffusion to discharge time": param.C_e,
+            "Ratio of solid diffusion to discharge time": param.C_p,
```
Comment (tinosulzer) [inline pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py:374]: you can print parameters to a file like this https://github.com/pybamm-team/PyBaMM/blob/d62c862d00845272e8111f91f9541f0e0c484dde/tests/unit/test_parameters/test_lithium_ion_parameters.py#L12-L15

**Comment by tinosulzer** [comment]
Tests won't run until conflicts are resolved

File: pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py (hunk 1 above)
Comment (molel-gt) [reply pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py:374]: Thanks.

Comment (tinosulzer) [reply pybamm/models/full_battery_models/lithium_ion/basic_dfn_half_cell.py:374]: So you don't need to add these to the list of variables

**Review by tinosulzer — APPROVED** [review]
Looks good, is this ready to merge once conflicts are resolved?

**Comment by molel-gt** [comment]
> Looks good, is this ready to merge once conflicts are resolved?

Yes.

**Comment by tinosulzer** [comment]
@all-contributors add @molel-gt for code, ideas

**Pull request merged**